### PR TITLE
feat(stepper): update component stepper with prop size and labelposition

### PR DESCRIPTION
**Pull request by JulienSaguez**
## Type

- Feature

#### What is expected?

update component stepper with prop size and labelposition

For the reference, check the figma https://www.figma.com/file/XHoO4V9LOFEFkAZsocYJ83/%5BCore%5D-Core?type=design&node-id=9924%3A119162&mode=design&t=KFPgzkAXzhRvPvYS-1 and the ticket https://jira.infra.online.net/browse/DS-587

### CODE REVIEW

File: packages/ui/src/components/Stepper/index.tsx [hunk 1]
```diff
@@ -8,6 +8,15 @@ import { Text } from '../Text'
 
 type Temporal = 'previous' | 'next' | 'current'
 
+const SIZES = {
+  small: {
+    height: 2,
```
Comment (DorianMaliszewski) [inline packages/ui/src/components/Stepper/index.tsx:13]: We should not have this hardcoded but comes from theme no ?

File: packages/ui/src/components/Stepper/index.tsx [hunk 2]
```diff
@@ -8,6 +8,15 @@ import { Text } from '../Text'
 
 type Temporal = 'previous' | 'next' | 'current'
 
+const SIZES = {
```
Comment (DorianMaliszewski) [inline packages/ui/src/components/Stepper/index.tsx:11]:
```suggestion
const LINE_HEIGHT_SIZES = {
```

better to understand the usage ?

**Comment by matthprost** [comment]
@JulienSaguez can you rebase your branch it won't trigger the deploy action until it is done, thanks 🙏

File: packages/ui/src/components/Stepper/index.tsx [hunk 3]
```diff
@@ -111,6 +132,8 @@ type StepperProps = {
   selected?: number
   children: ReactNode[]
   className?: string
+  labelPosition?: 'column' | 'row'
+  size?: 'small' | 'medium'
```
Comment (ModuloM) [inline packages/ui/src/components/Stepper/index.tsx:136]: I think you'll need two tests for this.

**Review by ModuloM — APPROVED** [review]

File: packages/ui/src/components/Stepper/index.tsx (hunk 3 above)
Comment (JulienSaguez) [reply packages/ui/src/components/Stepper/index.tsx:136]: Yes i can add test for the two props

**Review by PetmePant — CHANGES_REQUESTED** [review]
1) Need to change the prop options for labelPosition to bottom and right

2) Need to fix a small bug on the bullet number on small with right orientation, the text is not aligned center

Screenshot 2023-11-02 at 1 01 41 PM
Screenshot 2023-11-02 at 1 01 37 PM

**Pull request merged**